### ISSUE TITLE
a11y: VS Code intermittently inaccessible to screen reader after upgrade to 1.69.0

### ISSUE THREAD

**Issue by ndarilek** [microsoft/vscode#154398]
Immediately after upgrading to VS Code 1.69.0 on Windows 10, it intermittently stopped working with my screen reader. More specifically, starting VS Code usually doesn't present anything to NVDA. While I can OCR the window, I can't tab through or interact with controls and have them spoken. No other changes were made when this behavior started.

Sometimes it works, sometimes it doesn't. I haven't narrowed down a specific set of circumstances that makes it work vs. not. Sometimes it works/doesn't work with the exact same project. Restarting NVDA with the project loaded doesn't get it working.

I'm certain the project is loaded. It's Rust, so Rust Analyzer chugs away for several minutes. But the window remains inaccessible even after RA finishes its churn. And again, OCRing shows it working.

I've restarted NVDA and my PC multiple times.

This can't continue. Where can I get VS Code 1.68, and if I install it, how do I keep it from auto-upgrading to this latest version? And are there any other details I can provide? Hopefully someone else can duplicate this.

**Comment by ndarilek** [comment]
Update: I managed to open my main project and get an accessible window. Thinking I could outsmart the damned thing if I just left the window open, I alt-tabbed away. When I returned later, the window was no longer accessible. So the problem seems to affect existing windows as well. No idea if an inaccessible window might suddenly become accessible.

**Comment by lramos15** [comment]
Sorry to hear that, while we investigate this bug further if you wish to roll back to a previous version you may do so
Here's the windows download link for 1.68.1 https://update.code.visualstudio.com/1.68.1/win32-x64-user/stable you will have to add `"update.mode": "none"` to your settings to prevent auto update.

**Comment by ndarilek** [comment]
Update 2: Figured out   how to downgrade, installed 1.68.1, went into settings, disabled auto-update, then reinstalled 1.68.1 a second time because in the time it took me to disable auto-updates I was re-upgraded to 1.69. I did confirm that 1.69.0 still remained inaccessible by relaunching it before the update. So far 1.68.1 came up twice, and both times it was accessible.

For the moment things are fine. I'll stay on 1.68.1 until this is either resolved or needs additional information from me.

Thanks.

**Comment by jvesouza** [comment]
I know we're talking about different environments, but I haven't been able to reproduce this issue on my machine running fedora linux.
It seems to me that it would be interesting to test using the insider version of vscode, as long as possible, in order to verify if the problem is also present in that version.

**Comment by brian-upshaw** [comment]
I use Insiders on my home machine and the stable at work.  As of this morning my stable version auto-updated and this is now not working in the same fashion as the insiders build.  From the issue on the insiders branch #152291, the "--disable-features=CalculateNativeWinOcclusion" suggested workaround has not helped.  I'll be downgrading until this is fixed.

**Comment by deepak1556** [comment]
Merging to https://github.com/microsoft/vscode/issues/152291, please follow the issue for updates.

In the meantime, you can revert to `1.68` as mentioned in this comment https://github.com/microsoft/vscode/issues/154398#issuecomment-1178201905
